Withings-SDK-iOS/WithingsMeasureAPIClient.m

```diff
@@ -234,8 +261,18 @@ + (MeasuresDateRange*)dateRangeBetweenStartDate:(NSDate*)startDate andEndDate:(N
 {
     NSDate *startDate = _startDate ? _startDate : [NSDate dateWithTimeIntervalSince1970:0];
     NSDate *endDate = _endDate ? _endDate : [NSDate date];
-    return @{@"startdate" : @([startDate timeIntervalSince1970]),
```

Comment:
@jdrevet This is to satisfy the requirements specified in the Withings API documentation: http://oauth.withings.com/api/doc#api-Measure-get_sleep - "(recommended value : set to Noon in the user time zone)"